jwql/tests/test_preview_image.py

```diff
@@ -51,26 +49,27 @@ def test_directory(test_dir=TEST_DIRECTORY):
     """
     os.mkdir(test_dir)  # creates directory
     yield test_dir
-    print("teardown test directory")
     if os.path.isdir(test_dir):
         os.rmdir(test_dir)
 
 
+@pytest.mark.skipif(os.path.expanduser('~') == '/home/jenkins',
+                    reason='Requires access to central storage.')
 def test_make_image(test_directory):
```

Comment:
Do you need to add an `xfail` decorator here for Travis tests?

Reply:
@bhilbert4 I don't believe that `xfail` is needed here, as we would now like these tests to run and pass on the private jenkins server.  Using `skipif` here will skip the test entirely on the public jenkins server, which wouldn't be able to pass the test (as it requires access to central storage).

I'm not sure why are you mentioning Travis here(?) as we don't use Travis.

Reply:
Sorry. Too many repos with too many different CI testing setups to keep track of...

Reply:
😆 no worries